nilearn/image/image.py

```diff
@@ -1267,7 +1267,7 @@ def clean_img(
 
     # Clean signal
     clean_kwargs = {
-        k[7:]: v for k, v in kwargs.items() if k.startswith("clean__")
+        k[7:]: v for k, v in kwargs['kwargs'].items() if k.startswith("clean__")
```

Comment:
I don't understand this change. This will lead to a KeyError. I suggest reverting it

Reply:
I misunderstood how to pass the kwargs into the function. I was using kwargs={}, so I needed to add that change to find the kwarg.